database.go

```diff
@@ -68,9 +69,10 @@ type database struct {
 	tfl threatsForLookup
 	ml  sync.RWMutex // Protects tfl, err, and last
 
-	err             error     // Last error encountered
-	last            time.Time // Last time the threat list were synced
-	updateAPIErrors uint      // Number of times we attempted to contact the api and failed
+	err             error         // Last error encountered
+	errCh           chan struct{} // Used for waiting until not in an error state.
```

Comment:
Really strange name given that this indicates when its *not* in an error state. Perhaps `readyCh`?